src/main/java/mate/academy/springboot/swagger/controller/ProductController.java

```diff
@@ -0,0 +1,130 @@
+    @ApiOperation("Save a new product to the database")
+    public ProductResponseDto create(@RequestBody ProductRequestDto productRequestDto) {
+        Product product = productMapper.toModel(productRequestDto);
+        return productMapper.toDto(productService.save(product));
+    }
+
+    @GetMapping("/{id}")
+    @ApiOperation("Get product by id")
+    public ProductResponseDto getById(@PathVariable Long id) {
+        return productMapper.toDto(productService.getById(id));
+    }
+
+    @PutMapping("/{id}")
+    @ApiOperation("Update a product")
+    public ProductResponseDto update(@PathVariable Long id,
+                                     @RequestBody ProductRequestDto productRequestDto) {
+        Product product = productMapper.toModel(productRequestDto);
+        product.setId(id);
+        productService.save(product);
+        return productMapper.toDto(product);
+    }
+
+    @DeleteMapping("/{id}")
+    @ApiOperation("Delete a product by id")
+    @ResponseStatus(HttpStatus.NO_CONTENT)
+    public void delete(@PathVariable Long id) {
+        productService.deleteById(id);
+    }
+
+    @GetMapping("/by-price")
+    @ApiOperation("Get products by range of price")
+    public List<ProductResponseDto> findAllByPriceBetween(
+            @RequestParam
+            BigDecimal from,
+            @RequestParam
+            BigDecimal to,
+            @RequestParam (defaultValue = "2")
+            @ApiParam(value = "default value is '3'")
+            Integer count,
+            @RequestParam (defaultValue = "0")
+            @ApiParam(value = "default value is '1'")
+            Integer page,
+            @RequestParam (defaultValue = "id")
+            String sortBy) {
+        Sort sort = Sort.by(sortUtil.getOrders(sortBy));
+        PageRequest pageRequest = PageRequest.of(page, count, sort);
+        return productService.findAllByPriceBetween(from, to, pageRequest)
+                .stream()
+                .map(productMapper::toDto)
+                .collect(Collectors.toList());
+    }
+
+    @GetMapping()
+    public List<ProductResponseDto> getAll(@RequestParam (defaultValue = "2")
+                                               @ApiParam(value = "default value is '3'")
+                                               Integer count,
+                                           @RequestParam (defaultValue = "0")
+                                                @ApiParam(value = "default value is '1'")
+                                                Integer page,
+                                           @RequestParam (defaultValue = "id") String sortBy) {
+        Sort sort = Sort.by(sortUtil.getOrders(sortBy));
+        PageRequest pageRequest = PageRequest.of(page, count, sort);
+        return productService.findAll(pageRequest)
+                .stream()
+                .map(productMapper::toDto)
+                .collect(Collectors.toList());
+    }
+
+    @GetMapping("/inject")
+    public String inject() {

```

Comment:
It's up to you, but I think once you have liquibase, using the injection method is redundant